docs/content/introduction/cli/_index.md

```diff
@@ -4,5 +4,45 @@ weight = 4
 
 +++
 
-TODO
 
+## Download
+
+You'll find last release of `cdsctl` on [Github Releases](https://github.com/ovh/cds/releases/latest).
+
+## Commands
```

Comment:
it will be painful if we had to copy and paste the cdsctl help every time we change it

Reply:
this will be auto-generated, next PR